test/Test_Observer.dart

```diff
@@ -1,103 +1,93 @@
 
-class Test_Observer
-{
-  _tests()
-  {
-    group('Observer', ()
-    {
+class Test_Observer {
+  _tests() {
+    group('Observer', () {
       test('Constructor null args', () {
         // Create a blind Observer
-        mvc.IObserver observer = new mvc.Observer( null, null );
+        mvc.IObserver observer = mvc.Observer(() {}, null);
 
         // Make sure Observer was created
-        expect( observer, isNotNull );
+        expect(observer, isNotNull);
       });
 
       test('Constructor complete args', () {
         // Create an observer with callback method and caller instance defined
-        mvc.IObserver observer = new mvc.Observer( this.observerTestMethod, this );
+        mvc.IObserver observer = mvc.Observer(this.observerTestMethod, this);
 
         // Make sure the Observer was created
-        expect( observer, isNotNull  );
+        expect(observer, isNotNull);
 
         // Make sure caller and callback were set
-        expect( observer.getNotifyMethod(), isNotNull  );
-        expect( observer.notifyMethod, isNotNull  );
-        expect( observer.getNotifyContext(), isNotNull  );
-        expect( observer.notifyContext, isNotNull  );
-
+        expect(observer.getNotifyMethod(), isNotNull);
+        expect(observer.getNotifyContext(), isNotNull);
       });
 
       test('Constructor minimum args', () {
-        // Create an Observer with callack only
-        mvc.IObserver observer = new mvc.Observer( this.observerTestMethod );
+        // Create an Observer with callback only
+        mvc.IObserver observer = mvc.Observer(this.observerTestMethod);
 
         // Make sure Observer was created
-        expect( observer, isNotNull  );
+        expect(observer, isNotNull);
 
         // Make sure callback method was set
-        expect( observer.getNotifyMethod(), isNotNull );
+        expect(observer.getNotifyMethod(), isNotNull);
       });
 
       test('setNotifyMethod()', () {
         // Create an Observer
-        mvc.IObserver observer = new mvc.Observer( null );
+        mvc.IObserver observer = mvc.Observer(null);
 
         // Call setNotifyMethod()
-        observer.setNotifyMethod( observerTestMethod );
+        observer.setNotifyMethod(observerTestMethod);
 
         // Make sure callback method was set
-        expect( observer.getNotifyMethod(), isNotNull );
-        expect( observer.notifyMethod, isNotNull );
-
+        expect(observer.getNotifyMethod(), isNotNull);
       });
 
       test('setNotifyContext()', () {
         // Create an Observer
-        mvc.IObserver observer = new mvc.Observer( null );
+        mvc.IObserver observer = mvc.Observer(null);
 
         // Call setNotifyContext()
-        observer.setNotifyContext( this );
+        observer.setNotifyContext(this);
 
         // Make sure caller was set
-        expect( observer.getNotifyContext(), isNotNull );
-        expect( observer.getNotifyContext(), same(this) );
-        expect( observer.notifyContext, isNotNull );
```

Comment:
I removed direct access to the value of `notifyContext` because I though its excess and creates ambiguity in understanding of their use (as we already have methods). But I reverted the remove, and the test as well.